misk/src/main/kotlin/misk/web/WebConfig.kt

```diff
@@ -91,7 +91,9 @@ data class WebSslConfig(
   val port: Int,
   val cert_store: CertStoreConfig,
   val trust_store: TrustStoreConfig? = null,
-  val mutual_auth: MutualAuth = MutualAuth.REQUIRED
+  val mutual_auth: MutualAuth = MutualAuth.REQUIRED,
+  /** When set to true, only allow modern ciphers and protocols (as of this writing, TLS v1.3+). */
+  val modern_protocols_only: Boolean = false,
```

Comment:
I like the term modern, but only if it’s consistent with `modern` from the Google Cloud site.

Reply:
(Their definition of modern will change over time, as should ours!)